src/reactviews/pages/QueryResult/queryResultUtils.ts

```diff
@@ -30,3 +31,15 @@ export function hasResultsOrMessages(state: QueryResultWebviewState): boolean {
         state.messages.length > 0
     );
 }
+
+export const splitMessages = (messages: qr.IMessage[]): qr.IMessage[] => {
```

Comment:
Could you add a comment explaining this function? I'm not super familiar with the .split function syntax with /r & /n

Reply:
added